index.html

```diff
@@ -3308,10 +3312,16 @@
         Kanai, Tobie Langel, Tomoyuki Shimizu, Travis Leithead, and Wayne Carr
         for help with editing, reviews and feedback to this draft.
       </p>
+      <p>
+        <em>AirPlay</em>, <em>HDMI</em>, <em>Chromecast</em>, <em>DLNA</em> and
```

Comment:
@tidoust, is the recommendation to explicitly call these out in the spec, or does the "Non-W3C Trademarks; Member Trademarks" section in https://www.w3.org/Consortium/Legal/2002/trademarks-20021231 cover specs as well? I assume the term "Site" in this generic boilerplate refers to any resource hosted on the w3.org domain, including specs in the TR space.